src/rules/no-tags.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash');
+var rule = 'no-tags';
+var availableConfigs = {
+  'tags': []
+};
+
+function noTags(feature, fileName, configuration) {
+  var forbiddenTags = configuration.tags;
+
+  var featureErrors = checkTags(feature, forbiddenTags);
+
+  var childrenErrors = _(feature.children).map(function(child) {
+    return checkTags(child, forbiddenTags);
+  }).flatten().value();
+
+  return featureErrors.concat(childrenErrors);
+}
+
+function checkTags(node, forbiddenTags) {
+  return (node.tags || []).filter(function(tag) {
+    return isForbidden(tag, forbiddenTags);
+  }).map(function(tag) {
+    return createError(node, tag);
+  });
+}
+
+function isForbidden(tag, forbiddenTags) {
+  return _.includes(forbiddenTags, tag.name.replace(/@/, ''));
```

Comment:
OK. I was actually considering it, but wasn't really sure. Lets go for the simpler direction.